ui/js/dfv/src/admin/edit-pod/main-tabs/field-group.scss

```diff
@@ -1,10 +1,24 @@
 @import '../edit-pod-variables';
 
+$outline-color: #4f94d4;
```

Comment:
Should this be moved into the file `ui/js/dfv/src/admin/edit-pod/_edit-pod-variables.scss` along with the other variables?

Reply:
Looks like we already have a blue color we can use as outline `$color--wp-blue: #007CBA;` inside `ui/js/dfv/src/admin/edit-pod/_edit-pod-variables.scss`, I will use this one instead and get rid of the one I added.